packages/frontend/src/features/reporting/components/AdjudicateNotification/AdjudicateButton.tsx

```diff
@@ -1,6 +1,8 @@
 import { ReactComponent as ArrowRight } from '@/assets/svg/arrow-right.svg'
 import { ReactComponent as GavelRaisedIcon } from '@/assets/svg/gavel-raised.svg'
 import { ReactComponent as CloseIcon } from '@/assets/svg/close-button.svg'
+import { motion, useReducedMotion } from 'framer-motion'
+import { useMediaQuery } from 'react-responsive'
```

Comment:
I suggest to use this hook `useMedia` in `react-use` to check device width in order to reduce dependencies.

Reply:
fixed